cmd/memos.go

```diff
@@ -114,6 +114,77 @@ var (
 			}
 		},
 	}
+
+	mvrssCmd = &cobra.Command{
+		Use:   "mvrss",
```

Comment:
Let's add a comment for the name, otherwise others may not understand the meaning and may be confused with RSS.

Reply:
I've added a comment here. By the way, in the `cobra.Command`'s `Short` field, there's already a description for `mvrss`.

```golang
	mvrssCmd = &cobra.Command{
		Use:   "mvrss", // `mvrss` is a shortened for 'means move resource'
		Short: "Move resource between storage",
		Run: func(cmd *cobra.Command, _ []string) {
```